elrond.md

```diff
@@ -1274,6 +1330,248 @@ TODO: Implement [reserved keys and read-only runtimes](https://github.com/Elrond
          </locals>
 ```
 
+### Managed Buffer Ops
+
+#### Managed Buffer Internal Instructions
+
+```k
+    syntax InternalInstr ::= #getBuffer ( idx : Int )
+ // ---------------------------------------------------------------
+    rule <instrs> #getBuffer(BUFFER_IDX) => . ... </instrs>
+         <bytesStack> STACK => {HEAP[BUFFER_IDX]}:>Bytes : STACK </bytesStack>
```

Comment:
This is another `lhs => #Bottom` rule.

Both `{}:>Sort` and `Map[KItem]` are partial functions, so you should probably add `requires BUFFER_IDX in_keys(HEAP) andBool isBytes(HEAP[BUFFER_IDX])`.